sycl/test/lit.cfg.py

```diff
@@ -161,47 +150,52 @@ def getDeviceCount(device_type):
         gpu_check_on_linux_substitute = "| FileCheck %s"
         if cuda:
             gpu_run_on_linux_substitute += " SYCL_BE=PI_CUDA "
+else:
+    lit_config.warning("GPU device not found")
 
 config.substitutions.append( ('%GPU_RUN_PLACEHOLDER',  gpu_run_substitute) )
 config.substitutions.append( ('%GPU_RUN_ON_LINUX_PLACEHOLDER',  gpu_run_on_linux_substitute) )
 config.substitutions.append( ('%GPU_CHECK_PLACEHOLDER',  gpu_check_substitute) )
 config.substitutions.append( ('%GPU_CHECK_ON_LINUX_PLACEHOLDER',  gpu_check_on_linux_substitute) )
 
-if cuda:
-    config.substitutions.append( ('%sycl_triple',  "nvptx64-nvidia-cuda-sycldevice" ) )
-else:
-    config.substitutions.append( ('%sycl_triple',  "spir64-unknown-linux-sycldevice" ) )
-
 acc_run_substitute = "true"
 acc_check_substitute = ""
 if getDeviceCount("accelerator")[0] and platform.system() == "Linux":
-    print("Found available accelerator device")
+    lit_config.note("Found available accelerator device")
     acc_run_substitute = " env SYCL_DEVICE_TYPE=ACC "
     acc_check_substitute = "| FileCheck %s"
     config.available_features.add('accelerator')
+else:
+    lit_config.warning("Accelerator device not found")
 config.substitutions.append( ('%ACC_RUN_PLACEHOLDER',  acc_run_substitute) )
 config.substitutions.append( ('%ACC_CHECK_PLACEHOLDER',  acc_check_substitute) )
 
 # PI API either supports OpenCL or CUDA.
-opencl = False
 if not cuda:
-    opencl = True
     config.available_features.add('opencl')
 
+if cuda:
+    config.substitutions.append( ('%sycl_triple',  "nvptx64-nvidia-cuda-sycldevice" ) )
```

Comment:
What if I have both GPU devices CUDA and OpenCL. Will I be able to run tests on both devices?

Reply:
If I understand correctly, yes, but you need two separate launches of the llvm-lit with different value of  `SYCL_BE` argument